packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneDataStores.spec.ts

```diff
@@ -602,6 +625,52 @@ describeNoCompat("GC data store tombstone tests", (getTestObjectProvider) => {
 			},
 		);
 
+		// If this test starts failing due to runtime is closed errors try first adjusting `sweepTimeoutMs` above
+		itExpects(
+			"Requesting tombstoned datastores succeeds with future gcTombstoneEnforcementValue",
+			[
+				// Interactive client's request that succeeds
+				{
+					eventName: "fluid:telemetry:ContainerRuntime:GC_Tombstone_DataStore_Requested",
+					category: "generic",
+					gcEnforcementDisabled: true,
+				},
+				{
+					eventName:
+						"fluid:telemetry:ContainerRuntime:GarbageCollector:SweepReadyObject_Loaded",
+				},
+			],
+			async function () {
+				// Note: The Summarizers in this test don't use the "future" GC option - it only matters for the interactive client
+				const { unreferencedId, summarizingContainer, summarizer } =
+					await summarizationWithUnreferencedDataStoreAfterTime(sweepTimeoutMs);
+				await sendOpToUpdateSummaryTimestampToNow(summarizingContainer);
+
+				// The datastore should be tombstoned now
+				const { summaryVersion } = await summarize(summarizer);
+				const container = await loadContainer(
+					summaryVersion,
+					true /* disableTombstoneFailureViaOption */,
+				);
+				await sendOpToUpdateSummaryTimestampToNow(container);
```

Comment:
This should not be required.

Reply:
Curious as to how this would look

Reply:
There's no need to update the summary timestamp to now as we are not summarizing :)